CI on main branch keeps failing with JIT off and `-We` on `test_sdm_breakup.py` with ThrustRTC

```
PySDM/dynamics/collisions/collision.py:277: in compute_gamma
    self.particulator.backend.compute_gamma(
PySDM/backends/impl_thrust_rtc/nice_thrust.py:12: in wrapper
    result = func(*args, **kwargs)
PySDM/backends/impl_thrust_rtc/methods/collisions_methods.py:842: in compute_gamma
    self.__compute_gamma_body.launch_n(
PySDM/backends/impl_thrust_rtc/test_helpers/fake_thrust_rtc.py:98: in launch_n
    result = self.__internal_python_method__(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

__python_n__ = 1, prob = array([nan], dtype=float32)
rand = array([0.85729367], dtype=float32), idx = array([1, 0])
multiplicity = array([1, 1]), cell_id = array([0, 0])
collision_rate_deficit = array([0])
collision_rate = array([-9223372036854775808])
is_first_in_pair = array([ True, False]), out = array([nan], dtype=float32)

>   ???
E   RuntimeWarning: overflow encountered in long_scalars

<string>:245: RuntimeWarning

---------- coverage: platform linux, python 3.9.16-final-0 -----------
Coverage XML written to file coverage.xml

=========================== short test summary info ============================
FAILED tests/unit_tests/dynamics/collisions/test_sdm_breakup.py::TestSDMBreakup::test_nonadaptive_same_results_regardless_of_dt[ThrustRTC-10.0] - RuntimeWarning: overflow encountered in long_scalars
```

e.g., here: https://github.com/open-atmos/PySDM/actions/runs/4420069626/jobs/7749297734

--- a/PySDM/backends/impl_numba/storage.py
+++ b/PySDM/backends/impl_numba/storage.py
@@ -51,11 +51,12 @@ class Storage(StorageBase):
         return result
 
     def __setitem__(self, key, value):
-        if hasattr(value, "data"):
-            self.data[key] = value.data
-        else:
-            self.data[key] = value
-        return self
+        raise TypeError("Use to_ndarray() explicitly")
+        # if hasattr(value, "data"):
+        #     self.data[key] = value.data
+        # else:
+        #     self.data[key] = value
+        # return self
 
     def __iadd__(self, other):
         if isinstance(other, Storage):
@@ -187,3 +188,9 @@ class Storage(StorageBase):
 
     def upload(self, data):
         np.copyto(self.data, data, casting="safe")
+
+    def fill(self, other):
+        if isinstance(other, Storage):
+            self.data[:] = other.data
+        else:
+            self.data[:] = other

--- a/PySDM/backends/impl_numba/storage.py
+++ b/PySDM/backends/impl_numba/storage.py
@@ -51,12 +51,11 @@ class Storage(StorageBase):
         return result
 
     def __setitem__(self, key, value):
-        raise TypeError("Use to_ndarray() explicitly")
-        # if hasattr(value, "data"):
-        #     self.data[key] = value.data
-        # else:
-        #     self.data[key] = value
-        # return self
+        if hasattr(value, "data"):
+            self.data[key] = value.data
+        else:
+            self.data[key] = value
+        return self
 
     def __iadd__(self, other):
         if isinstance(other, Storage):

--- a/PySDM/backends/impl_thrust_rtc/storage.py
+++ b/PySDM/backends/impl_thrust_rtc/storage.py
@@ -497,4 +497,17 @@ def make_storage_class(BACKEND):  # pylint: disable=too-many-statements
             Impl.divide_if_not_zero(self, divisor)
             return self
 
+        def fill(self, value):
+            if isinstance(value, Storage):
+                trtc.Copy(value.data, self.data)
+            else:
+                if isinstance(value, int):
+                    dvalue = trtc.DVInt64(value)
+                elif isinstance(value, float):
+                    dvalue = BACKEND._get_floating_point(value)
+                else:
+                    raise TypeError("Only Storage, int and float are supported.")
+                trtc.Fill(self.data, dvalue)
+            return self
+
     return Storage

--- a/PySDM/dynamics/collisions/breakup_efficiencies/constEb.py
+++ b/PySDM/dynamics/collisions/breakup_efficiencies/constEb.py
@@ -12,4 +12,4 @@ class ConstEb:
         self.particulator = builder.particulator
 
     def __call__(self, output, is_first_in_pair):
-        output[:] = self.Eb
+        output.fill(self.Eb)

--- a/PySDM/dynamics/collisions/breakup_fragmentations/always_n.py
+++ b/PySDM/dynamics/collisions/breakup_fragmentations/always_n.py
@@ -13,7 +13,7 @@ class AlwaysN:  # pylint: disable=too-many-instance-attributes
         self.nfmax = nfmax
 
     def __call__(self, nf, frag_size, u01, is_first_in_pair):
-        nf[:] = self.N
+        nf.fill(self.N)
         self.x_plus_y.sum(self.particulator.attributes["volume"], is_first_in_pair)
         self.vmax.max(self.particulator.attributes["volume"], is_first_in_pair)
         frag_size.sum(self.particulator.attributes["volume"], is_first_in_pair)

--- a/PySDM/dynamics/collisions/breakup_fragmentations/straub2010.py
+++ b/PySDM/dynamics/collisions/breakup_fragmentations/straub2010.py
@@ -47,7 +47,7 @@ class Straub2010Nf:
 
         # compute the dimensionless numbers and CW=CKE * We
         self.arrays["tmp"].sum(self.particulator.attributes["volume"], is_first_in_pair)
-        self.arrays["Sc"][:] = self.arrays["tmp"][:]  # TODO #976
+        self.arrays["Sc"].fill(self.arrays["tmp"])
         self.arrays["Sc"] **= 2 / 3
         self.arrays["Sc"] *= (
             self.const.PI * self.const.sgm_w * (6 / self.const.PI) ** (2 / 3)
@@ -64,12 +64,12 @@ class Straub2010Nf:
         self.arrays["CKE"] *= self.arrays["tmp2"]
         self.arrays["CKE"] *= self.const.rho_w
 
-        self.arrays["We"][:] = self.arrays["CKE"][:]  # TODO #976
+        self.arrays["We"].fill(self.arrays["CKE"])
         self.arrays["We"].divide_if_not_zero(self.arrays["Sc"])
 
-        self.arrays["CW"][:] = self.arrays["We"][:]  # TODO #976
+        self.arrays["CW"].fill(self.arrays["We"])
         self.arrays["CW"] *= self.arrays["CKE"]
-        self.arrays["CW"] /= si.joule * 1e-6  # convert to µJ
+        self.arrays["CW"] /= si.uJ
 
         self.arrays["gam"].max(self.particulator.attributes["radius"], is_first_in_pair)
         self.arrays["tmp"].min(self.particulator.attributes["radius"], is_first_in_pair)

--- a/PySDM/dynamics/collisions/coalescence_efficiencies/constEc.py
+++ b/PySDM/dynamics/collisions/coalescence_efficiencies/constEc.py
@@ -10,4 +10,4 @@ class ConstEc:
         self.particulator = builder.particulator
 
     def __call__(self, output, is_first_in_pair):
-        output[:] = self.Ec
+        output.fill(self.Ec)

--- a/PySDM/dynamics/collisions/coalescence_efficiencies/straub2010.py
+++ b/PySDM/dynamics/collisions/coalescence_efficiencies/straub2010.py
@@ -25,7 +25,7 @@ class Straub2010Ec:
 
     def __call__(self, output, is_first_in_pair):
         self.arrays["tmp"].sum(self.particulator.attributes["volume"], is_first_in_pair)
-        self.arrays["Sc"][:] = self.arrays["tmp"][:]
+        self.arrays["Sc"].fill(self.arrays["tmp"])
         self.arrays["tmp"] *= 2
 
         self.arrays["tmp2"].distance(
@@ -47,4 +47,4 @@ class Straub2010Ec:
         self.arrays["We"].divide_if_not_zero(self.arrays["Sc"])
         self.arrays["We"] *= -1.15
 
-        output[:] = np.exp(self.arrays["We"])
+        output[:] = np.exp(self.arrays["We"])  # TODO #976

--- a/PySDM/dynamics/collisions/collision_kernels/constantK.py
+++ b/PySDM/dynamics/collisions/collision_kernels/constantK.py
@@ -9,8 +9,7 @@ class ConstantK:
         self.particulator = None
 
     def __call__(self, output, is_first_in_pair):
-        output *= 0
-        output += self.a
+        output.fill(self.a)
 
     def register(self, builder):
         self.particulator = builder.particulator

